Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -166,6 +167,23 @@ def is_valid_type(_type):
 # ====================================================================================
 
 
+def add(num1, num2):
+    """
+    Adds two integers.
+
+    Known limitations:
+     - The sum of both numbers cannot exceed 300.
+     - May take some time to count.
+    :param num1: First number.
+    :param num2: Second number.
+    :return: result of two numbers
+    """
+    start = time()
+    sleep(num1)
```

Comment:
```suggestion
    sleep(num1*num2)
```

Reply:
Will add multiply function to address this usecase